src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+	private static final String DELIMITERS = ",|:";
+	private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+	private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile(CUSTOM_DELIMITER);
+
+	public static int splitAndSum(String input) {
+		if (isNullOrBlank(input)) {
+			return 0;
+		}
+		return sumNumbers(removeDelimiter(input));
+	}
+
+	private static String[] removeDelimiter(String input) {
+		Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(input);
+		if (m.find()) {
+			String customDelimiter = m.group(1);
+			return m.group(2).split(customDelimiter);
+		}
+		return input.split(DELIMITERS);
+	}
+
+	private static boolean isNullOrBlank(String input) {
+		return input == null || input.isEmpty();
+	}
```

Comment:
splitAndSum에서 
1. isNullOrBlank
2. removeDelimiter
3. sumNumbers
순으로 호출하고 있기 때문에 메서드 선언 위치도 동일하게 변경하면 읽기 좋은 코드가 될 것 같습니다.
```suggestion
	private static boolean isNullOrBlank(String input) {
		return input == null || input.isEmpty();
	}
	
	private static String[] removeDelimiter(String input) {
		Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(input);
		if (m.find()) {
			String customDelimiter = m.group(1);
			return m.group(2).split(customDelimiter);
		}
		return input.split(DELIMITERS);
	}
```